modules/MarketAnalysis.py

```diff
@@ -106,9 +174,17 @@ def get_rate_suggestion(self, cur, rates=None, method='percentile'):
             if len(rates) == 0:
                 return 0
             if method == 'percentile':
+                # rates is a tuple with the first entry being unixtime
+                rates = [x[1] for x in rates]
                 return self.get_percentile(rates, self.lending_style)
             elif method == 'golden_cross':
-                raise ValueError("{0} not yet implmented")
+                if len(rates) < 1800:
```

Comment:
The get_rate_suggesion method takes a method argument now, so if we want we can do that functionality there. Meaning we could catch the correct exceptions and run a different method if it makes sense. 

The default for the method is golden_cross, which I'd actually like to keep, but it will check for pandas / numpy and act accordingly.